src/gui/widgets/LcdWidget.cpp

```diff
@@ -39,42 +39,32 @@
 
 
 
-//! @todo: in C++11, we can use delegating ctors
-#define DEFAULT_LCDWIDGET_INITIALIZER_LIST \
-	QWidget( parent ), \
-	m_label(), \
-	m_textColor( 255, 255, 255 ), \
-	m_textShadowColor( 64, 64, 64 )
-
 LcdWidget::LcdWidget( QWidget* parent, const QString& name ) :
-	DEFAULT_LCDWIDGET_INITIALIZER_LIST,
-	m_numDigits( 1 )
+	LcdWidget( 1, parent, name)
 {
-	initUi( name );
 }
 
 
 
 
 LcdWidget::LcdWidget( int numDigits, QWidget* parent, const QString& name ) :
-	DEFAULT_LCDWIDGET_INITIALIZER_LIST,
-	m_numDigits( numDigits )
+	LcdWidget( numDigits, QString("19green"), parent, name )
```

Comment:
Since you've added the default color here, would you mind removing it from the default params of `initUi` in the header?